pki/src/test/resources/keystore/invalidpartner1client1/crl.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN X509 CRL-----
```

Comment:
I’d rather we generate these during testing.

Reply:
Same for all these files. It woild be easier to maintain going forward.

Reply:
Idk if generating them dynamically provides that much value. I'm sure these certificates will have valid date long enough that we won't have an issue maintaining them.

What are your maintenance concerns? Just the expiry date or something else?

Reply:
How were they generated? How are they matched? What are their relationships?
Some of those certificate files are combined certs - a whole chain. From experience, you don’t just conjure those easily.

Reply:
The scripts to generate them are available here https://github.com/perusworld/besu-isolated-networks/tree/p2p-over-ssl/scripts, for the unit tests this script was used - gen-test-keys.sh. I can add those scripts to this module itself if that helps. We are working on adding support for pkcs11/12/jks keystores to use it in Node Signing, TLS support p2p. So we generating more keys for acceptance tests and dev quick start etc. Due to the complexity I would reccommend to use the scripts. Thanks

Reply:
Please include instructions on how to regenerate those files at a minimum.

Reply:
Your repository has no license. Please add a license.

Reply:
thanks, i have updated accordingly